arcsi/model/user.py

```diff
@@ -27,3 +28,20 @@ class User(db.Model, UserMixin):
 
     def __repr__(self):
         return "<Host {}>".format(self.name)
+
+class User2(db.Model, SQLAlchemyUserMixin):
```

Comment:
This is a good solution to independently work with and develop the flask based authentication and authorisation! 

Are these columns defined by you or by some template? 

Because it seems like compared to the `User` model we are currently using the only addition by praetorian is `is_active` the only addition by us are the `butt_u` `butt_p` `shows` columns everything else mandatory is the same. So there should be no problem transforming current users then merging it back into User! 

Good job!

Reply:
Yes, I had some problems with the flask-security and flask-praetorian needed fields and methods. First I tried to copy the original User model (with name and password for example) and give back the flask-praetorian needed fields as a function (username() and hashed_password() functions gave back the value of name and password fields) but it turned out that these fields need to be exactly defined as username and hashed_password.